frontend/src/scenes/settings/environment/SessionRecordingIngestionSettings.tsx

```diff
@@ -480,6 +508,7 @@ export function SessionRecordingIngestionSettings(): JSX.Element | null {
                 <FlaggedFeature flag={FEATURE_FLAGS.SESSION_REPLAY_URL_BLOCKLIST}>
                     <UrlBlocklistOptions />
                 </FlaggedFeature>
+                <EventTriggerOptions />
```

Comment:
we should maybe flag this as well?

can put it inside the URL_BLOCKLIST flag if you like

i think we'll just release all of these / at least release them together